tmt/steps/execute/run.sh

```diff
@@ -140,11 +159,16 @@ tmt_run_test () {
     tmt_verbose 2 "type: ${tmt_TYPE}"
 
     tmt_run_${tmt_TYPE} "$path" "$environment" "$cmd"
-    echo "$?" >"$tmt_LOGCODE_F"
+    local exitcode="$?"
```

Comment:
I cannot see why you need to save it to local variable. (I consider the previous implementation more fool-proof. If it fails to save to a file, the test is failed either way, right?

Could we also please avoid declaring locals in the middle of the code?

Reply:
Results handling functionality moved to the `simple` execute plugin.